ajax/libs/fuzzysort/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "fuzzysort",
+  "description": "Fast SublimeText-like fuzzy search for JavaScript",
+  "keywords": [
+    "fuzzy",
+    "search",
+    "filter",
+    "javascript",
+    "sublime"
+  ],
+  "author": "farzher",
+  "license": "MIT",
+  "filename": "fuzzysort.min.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/farzher/fuzzysort.git"
+  },
+  "autoupdate": {
```

Comment:
Let's use npm auto-update as the commit msg described. See https://github.com/cdnjs/cdnjs/blob/master/documents/autoupdate.md